api/taskcluster/webhook.go

```diff
@@ -167,7 +167,7 @@ func handleStatusEvent(ctx context.Context, payload []byte) (bool, error) {
 	} else {
 		sender := status.GetCommit().GetAuthor().GetLogin()
 		if sender != "" {
-			labels.Add(sender)
+			labels.Add("user:" + sender)
```

Comment:
There's existing test case for this in `webhook_test.go` (search for "master"). Please adjust the test accordingly to cover this as well.

Reply:
Added `"user:person"` to the labels we pass, and changed the spec for pending checks to not include anything other than the channel.